<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>636</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="FormSection" name="formSectionStyle" native="true"/>
   </item>
   <item>
    <widget class="FormSection" name="formSectionLevel" native="true"/>
   </item>
   <item>
    <widget class="FormSection" name="formSectionAdvanced" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widgetStyle" native="true">
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Окно предпросмотра</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonPreview1">
           <property name="text">
            <string>&amp;x1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPreview2">
           <property name="text">
            <string>x&amp;2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPreview4">
           <property name="text">
            <string>x&amp;4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonPreview">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Размер картинки в окне предпросмотра&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>info</string>
           </property>
           <property name="icon">
            <iconset resource="Japuzzle.qrc">
             <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetLevel" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Профиль</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBoxLevelProfile"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonLevelProfile">
          <property name="toolTip">
           <string>Можно выбрать профиль или самому настроить доступность тех или иных элементов, автоматизирующих (упрощающих) решение задач</string>
          </property>
          <property name="text">
           <string>info</string>
          </property>
          <property name="icon">
           <iconset resource="Japuzzle.qrc">
            <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Подсветка строк</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonHighlightOn">
           <property name="text">
            <string>Включена</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonHighlightOff">
           <property name="text">
            <string>Выключена</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonHighlight">
           <property name="toolTip">
            <string>Подсветка строки и столбца, на которые указывает курсор</string>
           </property>
           <property name="text">
            <string>info</string>
           </property>
           <property name="icon">
            <iconset resource="Japuzzle.qrc">
             <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Компактные цифры</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonCompactDigitsOn">
           <property name="text">
            <string>Включена</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxCompactDigits">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>32</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonCompactDigitsOff">
           <property name="text">
            <string>Выключена</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonCompactDigits">
           <property name="toolTip">
            <string>Сокращать количество показваемых цифр, если невозможно отобразить все</string>
           </property>
           <property name="text">
            <string>info</string>
           </property>
           <property name="icon">
            <iconset resource="Japuzzle.qrc">
             <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Раскраска цифр</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonDigitsSmart">
           <property name="text">
            <string>Умный</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonDigitsAuto">
           <property name="text">
            <string>Простой</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonDigitsManual">
           <property name="text">
            <string>Вручную</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonDigits">
           <property name="toolTip">
            <string>Как будут раскрашиваться цифры:
- &quot;Умный&quot; алгоритм раскраски
- &quot;Простой&quot; алгоритм
- Раскраска цифр &quot;вручную&quot;
Яркий цвет - цифра не закрыта, блёклый - закрыта, красный - ошибка
</string>
           </property>
           <property name="text">
            <string>info</string>
           </property>
           <property name="icon">
            <iconset resource="Japuzzle.qrc">
             <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Всплывающее окно</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonCalcWindowSmart">
           <property name="text">
            <string>Умное</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonCalcWindowSimple">
           <property name="text">
            <string>Простое</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonCalcWindowNo">
           <property name="text">
            <string>Нет</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonCalcWindow">
           <property name="toolTip">
            <string>Всплывающее окно при множественном задании ячеек:
- &quot;Умный&quot; алгоритм подсказки
- &quot;Простой&quot; алгоритм подсказки
- не выводить подсказку
</string>
           </property>
           <property name="text">
            <string>info</string>
           </property>
           <property name="icon">
            <iconset resource="Japuzzle.qrc">
             <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Сложность рисунка</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonCalcStarsAuto">
           <property name="text">
            <string>Автоматически</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonCalcStarsManual">
           <property name="text">
            <string>Вручную</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonCalcStars">
           <property name="toolTip">
            <string>Автоматическое определение сложности рисунка (звёзд)</string>
           </property>
           <property name="text">
            <string>info</string>
           </property>
           <property name="icon">
            <iconset resource="Japuzzle.qrc">
             <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetAdvanced" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Лимит отмены</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinBoxUndoLimit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string> Мб</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>512</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonInfoStackLimit">
          <property name="toolTip">
           <string>Ограничение размера памяти, выделяемой на на очередь отмены.</string>
          </property>
          <property name="text">
           <string>info</string>
          </property>
          <property name="icon">
           <iconset resource="Japuzzle.qrc">
            <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Авто сохранение</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxAutoSave">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> мин</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>30.000000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxAutoSaveDisable">
          <property name="text">
           <string>Не сохранять</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonInfoAutoSave">
          <property name="toolTip">
           <string>Автоматическое сохранение текущего рисунка через указанные интервалы времени.</string>
          </property>
          <property name="text">
           <string>info</string>
          </property>
          <property name="icon">
           <iconset resource="Japuzzle.qrc">
            <normaloff>:/Icons/Info.png</normaloff>:/Icons/Info.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormSection</class>
   <extends>QWidget</extends>
   <header location="global">Lib/CoreUi/FormSection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Japuzzle.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
